cgroup_top.py

```diff
@@ -100,6 +103,60 @@ def cmd_exists(cmd):
 
 ## Utils
 
+
+def strip_prefix(prefix, text):
+    if text.startswith(prefix):
+        return text[len(prefix):]
+    return text
+
+
+def docker_container_name(container_id, default, cache=dict()):
+    # Python’s default arguments are evaluated when the function is
```

Comment:
Funny UTF-8 sequence here :) Can you use regular ``'`` in "Python's" ? In vim it displays as a nice ``# Python�~@~Ys``. Other ones are OK.